core/src/splitting/InstrumentalSplittingRule.cpp

```diff
@@ -16,6 +16,7 @@
  #-------------------------------------------------------------------------------*/
 
 #include <algorithm>
+#include <cmath> // TODO header includes consistency: this is not needed in RegressionSplitting (std::isnan)
```

Comment:
I meant that RegressionSplitting.cpp compiles without including cmath, but not InstrumentalSplittingRule.cpp, because the former has some extra includes in RegressionSplitting.h that InstrumentalSplittingRule.h does not, even thought the required functionality from imports is the same.

I can file a "code quality" issue to do a consistency "checkup" later?

Reply:
👍 a new issue sounds good.